.gitignore

```diff
@@ -2,6 +2,6 @@
 /docs/src/AbstractAlgebra/
 /docs/src/Hecke/
 /docs/src/Nemo/
-/docs/src/experimental/
+/docs/src/Experimental/
```

Comment:
This is a pitfall for everyone still using the "old" experimental structure. They probably miss adding their docs files to git due to this. Unfortunately, I have no good alternative (apart from everyone re-structuring their experimental code).

Reply:
It is only until the old stuff is gone and it is detected by github actions.